docs/index_es.html

```diff
@@ -0,0 +1,118 @@
+          <li>Están dispuesto/as a admitir cuando están equivocado/as y no tienen miedo de decir "No lo sé".</li>
+          <li>Puede que le guste o no escribir documentación, pero lo hace de todos modos para futuro/as mantenedore/as.</li>
+          <li>Puede que le guste o no escribir pruebas, pero trata de aprender a hacerlo si el equipo o proyecto lo necesita.</li>
+          <li>Agradece a lo/as demás por su tiempo, esfuerzo y energía.</li>
+          <li>Puede tener fondos de escritorio coloridos.</li>
+          <li>Apoya código en producción, incluso si no lo escribieron.</li>
+          <li>Puede sentirse como un/a impostor/a a veces, y entiende que otro/as también pueden hacerlo.</li>
+          <li>Cree que todo/as en la sala son tan inteligentes y capaces como ello/as.</li>
+          <li>Ayudará a subir de nivel a otro/as y pedirá ayuda cuando la necesiten.</li>
+          <li>Nunca deja de aprender, pero puede sentirse totalmente abrumado/a por la cantidad de aprendizaje que hay que hacer.</li>
+          <li>Trata de mantener las discusiones productivas y permite que otro/as expresen su opinión antes de que el equipo tome una decisión.</li>
+          <li>Está dispuesto/a a abandonar su zona de confort..</li>
+          <li>Está dispuesto a dejar sus contribuciones a la comunidad a su manera cuando es posible y aprecia las formas en que otro/as contribuyen cuando pueden sentirse cómodo/as..</li>
+          <li>Puede ser un/a programador/a lento/a.</li>
+          <li>Tiene días productivos e improductivos.</li>
+          <li>No se toma demasiado en serio.</li>
+          <li>Dice: "Nunca he oído hablar de eso", en lugar de asentir y fingir.</li>
+          <li>Es confiable.</li>
+          <li>Trabaja para vivir, en lugar de vivir para trabajar.</li>
+          <li>A veces pierde su trabajo.</li>
+          <li>No es necesario tener todo el código base memorizado.</li>
+          <li>Respeta y defiende los códigos de conducta de la comunidad.</li>
+          <li>Pueden trabajar desde casa, la oficina, una cafetería o donde mejor funcione para ello/as.</li>
+          <li>No odia las herramientas, los procesos o los lenguajes que preferiría no usar o que otro/as usan.</li>
+          <li>No está definido por la computadora que están usando.</li>
+          <li>Puede decorar su computadora portátil y espacio de trabajo de la forma que desee, y también es respetuoso/a con la decoración de lo/as demás (o la falta de estas).</li>
+          <li>No está definido/a por tormentas de Tweets miopes por capitalistas de riesgo (VC) despistado/as.</li>
+          <li>No ridiculiza profesiones enteras dentro de la ingeniería, especialmente cuando se encuentra en una posición de liderazgo.</li>
+      </ul>
+    </section>
+      <section>
+        <p>
+          ¿Notas que falta algo en la lista? Lo/as ingenieros 1x suelen ser humildes y están dispuesto/as a aceptar solicitudes de incorporación de cambios para corregir errores.
```

Comment:
Most of the text is written in "usted" conjugations, so for consistency:

```suggestion
          ¿Nota que falta algo en la lista? Lo/as ingenieros 1x suelen ser humildes y están dispuesto/as a aceptar solicitudes de incorporación de cambios para corregir errores.
```